### ISSUE TITLE
Profile Picture - Previously Reported

### ISSUE THREAD

**Issue by jmartins1** [grokability/snipe-it#15076]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

Reported an issue earlier where the default profile picture was missing and a placeholder was shown instead.  Previous issue below.

![issue](https://github.com/user-attachments/assets/7101d8d4-ddb3-4659-9cec-5d3133c12133)

Although the update added a default profile picture, an issue remains.

If an admin attached a picture to the user's profile and that picture was then removed/deleted but not replaced, SnipeIT will display a picture placeholder instead of defaulting back to the system default profile picture

Please note I manually removed the profile picture in the image below to illustrate the issue. I had to manually assign a profile picture to correct the issue.

![Asset](https://github.com/user-attachments/assets/1282e9ce-01b4-404f-856a-2520bc05c5b0)

### Reproduction steps

1. View an asset check out to page.
2.
3.
...

### Expected behavior

A profile picture should always be available. Whether it be an admin assigned picture or a system default picture. It should never display a placeholder.

### Snipe-IT Version

7.0.9

### Operating System

Server 2022

### Web Server

WAMP

### PHP Version

8.2.5

**Comment by snipe** [comment]
> If an admin attached a picture to the user's profile and that picture was then removed/deleted but not replaced

How are you removing that picture?

> SnipeIT will display a picture placeholder instead of defaulting back to the system default profile picture

I'm not exactly sure what you mean by "a picture placeholder"

**Comment by snipe** [comment]
I'm not able to reproduce this (if I'm understanding the issue correctly.) :(

**Comment by snipe** [comment]
In testing this, I did find a potential issue where the default system image got deleted from the disk if you click on "delete avatar", which gets a little weird, since that file is currently tracked in git, so the actual delete causes a git change where it shows the file as deleted. That's not awesome, but it's sort of expected, since we have to track the default in git in order to make sure users have it by default.

**Comment by jmartins1** [comment]
Thank you. Here's the issue a little bit deeper. By default, all users receive a default profile picture. Great, but later, if you assign them a personal profile picture, there is no way to default back to the system default without manually assigning the system default image. If you delete the picture from the avatar directory, then it breaks the link to the profile, and you have a missing image and end up having to manually assign the system default image.


What I'm suggesting is that if a user had a picture assigned to them (something other than the default system image) and then it is no longer in the avatar directory. SnipeIT should revert the user's profile picture to the system default automatically.

I can make a demo video if this doesn't make sense. Thanks for looking into this.



⁣Get BlueMail for Android ​

**Comment by snipe** [comment]
> What I'm suggesting is that if a user had a picture assigned to them (something other than the default system image) and then it is no longer in the avatar directory. SnipeIT should revert the user's profile picture to the system default automatically.

It does do that, but if you first deleted the one that comes with Snipe-IT, it would actually delete that file off the disk, which could result in the broken image.

**Comment by jmartins1** [comment]
Unfortunately, that's not my experience. I assigned a user a profile picture; just testing. I then wanted to revert back to the system default. I didn't see an option to remove the picture, so I deleted the image I assigned from the avatar directory. This broke the profile image, as I mentioned in the original post. Understandably, because the DB is looking for the file and it's no longer there. SnipeIT did not replace the image. It just left it with a missing image. To fix this, I had to manually assign the system default to the profile.  

⁣Get BlueMail for Android ​
